src/js/Mixins/Modes/Mode.Removal.js

```diff
@@ -16,7 +16,7 @@ const GlobalRemovalMode = {
   enableGlobalRemovalMode() {
     const isRelevant = layer =>
       layer.pm &&
-      !(layer.pm.options && layer.pm.options.preventMarkerRemoval) &&
+      !((layer instanceof L.Marker || layer instanceof L.CircleMarker) && layer.pm.options && layer.pm.options.preventMarkerRemoval) &&
```

Comment:
Can you add a comment here explaining when a layer is relevant?
I would also like to see a test for this - otherwise it will be quite hard to debug when this causes a bug or unexpected behaviour in the future

Reply:
I re-thinked about that ... Why is it checking for `preventMarkerRemoval`? Because it only adds a `click` event and that is allowed for all Markers no matter if the option `preventMarkerRemoval` is passed. 

`preventMarkerRemoval` only restrict that a Marker can't deleted by right clicking (contextmenu).

My suggestion is to remove the complete line.

Reply:
You mean the `preventMarkerRemoval` option has no effect here right now? I think you are right, in which case this can be deleted.